book/website/communication/peer-review/peer-review-open.md

```diff
@@ -0,0 +1,50 @@
+(cm-pr-open)=
+# Open Peer Review
+
+## What is Open Peer Review? 
+
+Open Peer review can refer to various practises, including signing your review, publishing reviews along with the paper, and allowing for the community to contribute to the peer review process ([Open Science Community Utrecht](https://openscience-utrecht.com/peer-review)). Below some different types of Open Peer Review are highlighted, as well as the benefits of opening up the peer review process.
+
+(cm-pr-open-what)=
+## Different types of Open Peer Review
+
+There are various [models of Open Peer review](https://plos.org/resource/open-peer-review/) (see {cite:ps}`RossHellauer2017openreview` for more details):
+
+- **Open Content**
+  - Publishing peer review content
+  - Decoupling the peer review process from the publication process (see [recent announcement by eLife](https://elifesciences.org/for-the-press/b2329859/elife-ends-accept-reject-decisions-following-peer-review))
+- **Open Participation**
+  - Open review before publication through preprints
+  - Post-publication commenting
+- **Open Identities** (author or reviewer)
+  - Open discussion between authors, editors and reviewers
+
+Similar to open science practices in general - open peer review is not an 'all or nothing' concept. 
+Instead, the open components that make the most sense can be introduced first - or the platform that you're using might not have opened up all of this yet.
+
+One of the more debated parts of Open Peer Review is whether reviews should be signed {cite:ps}`RossHellauer2017openreview` {cite:ps}`Bravo2019openreview`.
+Not everyone is comfortable signing their reviews - particularly people from minoritized groups in research {cite:ps}`Fox2021openreview`. 
+This could result in less critical reviews if revealing identity is a requirement of Peer Review {cite:ps}`Fox2021openreview`. 
+
+(cm-pr-open-benefits)=
+## Benefits of Open Peer Review
+
+- Including increasing accountability of reviewers, which should in turn increase the quality, fairness, constructiveness and courteousness of reviews 
+- Transparency allows for validation of peer review and [identifying biases](https://twitter.com/giladfeldman/status/1568391773353037824)
+- Recognition for Peer Review (especially if it can become a {ref}`Research Object that can be cited<cm-citable-cite>`Research Object that can be cited
```

Comment:
```suggestion
- Recognition for Peer Review (especially if it can become a {ref}`Research Object that can be cited<cm-citable-cite>`Research Object that can be cited)
```

Reply:
Actually that whole last bit needs to be deleted. Thanks for spotting!